CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * **no-result-notice** adjust margin for no result notice in mat-table.
 * **terra-taglist** add flex-wrap
 * **tooltip** now subscribes to router events in ngOnInit, not ngOnChanges and makes sure to only call `hide()` when the tooltip is shown.
+* **mat-table** remove cursor pointer from table row
```

Comment:
Die v5.5 wurde bereits released. Der Changelog muss in Zeile 6 ergänzt werden.

Reply:
ist geändert